Cookbook/Technical-Documents/SwiftGen.md

```diff
@@ -0,0 +1,34 @@
+## SwiftGen
+
+[SwiftGen](https://github.com/SwiftGen/SwiftGen) allows plenty of possibilities to automatically generate code based on resources we store in our project. Following sections will describe which types of resources are currently integrated with SwiftGen and how to use them.
+
+### Image assets
+
+Using icons and images with our SwiftGen integration is really easy. The only thing you have to do is to add the image to `BabylonDesignLibrary/DesignLibrary/Assets.xcassets` and then you can just use it by calling `appearance.tokens.icons.image(.myNewIcon)`. Before compilation of `BabylonDesignLibrary`, SwiftGen is executed and regenerates the identifiers.
```

Comment:
| then you can just use it...
doesn't it require first to build the app? We might want to have this as a Xcode extension in the future so that the build is not required, or at least to mention how to call SwiftGen manually from command line

Reply:
If you will use identifier without compiling first you will probably get an error from the Xcode but it will be removed during compilation and it won't fail.

Reply:
What you won't have will be autocompletion

Reply:
I can add that note